### ISSUE TITLE
Uncaught Error: Cannot find module 'standard-flow'

### ISSUE THREAD

**Issue by evansiroky** [ricardofbarros/linter-js-standard#166]
Not sure exactly how to reproduce, but it seems that this error happens when using the style setting of `standard-flow` and while the separate package `flow-ide` is in use.

**Atom**: 1.14.3 x64
**Electron**: 1.3.13
**OS**: Mac OS X 10.12.3
**Thrown From**: [linter-js-standard](https://github.com/ricardofbarros/linter-js-standard) package 3.9.0

### Stack Trace

Uncaught Error: Cannot find module 'standard-flow'

```
At /Applications/Atom.app/Contents/Resources/app.asar/src/module-cache.js:379

Error: Cannot find module 'standard-flow'
    at Module._resolveFilename (module.js:455:15)
    at Module._resolveFilename (/Applications/Atom.app/Contents/Resources/electron.asar/common/reset-search-paths.js:35:12)
    at Function.Module._resolveFilename (/app.asar/src/module-cache.js:383:52)
    at Function.Module._load (module.js:403:25)
    at Module.require (module.js:483:17)
    at require (/app.asar/src/native-compile-cache.js:50:27)
    at requireWithLocalOverride (/packages/linter-js-standard/lib/utils/select-style.js:16:12)
    at /packages/linter-js-standard/lib/utils/select-style.js:23:12
    at exports.allowUnsafeNewFunction (/packages/linter-js-standard/node_modules/loophole/lib/loophole.js:25:14)
    at pickStandard (/packages/linter-js-standard/lib/utils/select-style.js:22:10)
    at selectStyle (/packages/linter-js-standard/lib/utils/select-style.js:58:24)
    at Object.__cacheTextEditor (/packages/linter-js-standard/lib/init.js:110:17)
    at storeSettings (/packages/linter-js-standard/lib/init.js:79:14)
    at Function.module.exports.Emitter.simpleDispatch (/app.asar/node_modules/event-kit/lib/emitter.js:25:14)
    at Emitter.module.exports.Emitter.emit (/app.asar/node_modules/event-kit/lib/emitter.js:129:28)
    at /app.asar/src/pane-container.js:348:27
    at Function.module.exports.Emitter.simpleDispatch (/app.asar/node_modules/event-kit/lib/emitter.js:25:14)
    at Emitter.module.exports.Emitter.emit (/app.asar/node_modules/event-kit/lib/emitter.js:129:28)
    at Pane.module.exports.Pane.setActiveItem (/app.asar/src/pane.js:300:22)
    at Pane.module.exports.Pane.activateItem (/app.asar/src/pane.js:452:21)
    at TabBarView.module.exports.TabBarView.onMouseDown (/app.asar/node_modules/tabs/lib/tab-bar-view.js:678:19)
```

### Commands

```
     -5:52.9.0 docblockr:parse-enter (input.hidden-input)
     -5:52.9.0 editor:newline (input.hidden-input)
     -5:52.3.0 core:paste (input.hidden-input)
     -5:45.8.0 core:save (input.hidden-input)
     -0:30.3.0 core:backspace (input.hidden-input)
     -0:29.9.0 core:save (input.hidden-input)
     -0:10.7.0 core:copy (input.hidden-input)
```

### Non-Core Packages

```
atom-beautify 0.29.17 
atom-jade 0.3.0 
autocomplete-modules 1.6.7 
docblockr 0.9.1 
expand-region 0.2.6 
file-icons 2.0.15 
flow-ide 1.3.0 
jumpy 3.1.3 
language-svg 0.9.2 
linter 1.11.23 
linter-js-standard 3.9.0 
linter-write-good 0.8.0 
local-history 4.0.1 
manage-globals 1.0.0 
react 0.16.2 
react-snippets 0.7.0 
snippet-generator 0.3.2 
Stylus 3.1.0 
```

**Comment by hipertracker** [comment]
+1

**Comment by sonicdoe** [comment]
Fix released in v3.9.3.

linter-js-standard did not include standard-flow as a dependency which caused us to throw the “Cannot find module” error if standard-flow was not locally installed.
